src/features/ValueEditor/ValueEditor.tsx

```diff
@@ -47,19 +46,23 @@ const editorOptions: editor.IStandaloneEditorConstructionOptions = {
   renderLineHighlight: 'none',
 }
 
+interface Props {
+  value: Value
+  index: number
+  onAdd: () => void
```

Comment:
I extracted the props and made `onAdd` non-optional; you can always add a value editor and the prop will never be undefined